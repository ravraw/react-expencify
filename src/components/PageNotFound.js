import React from "react";
import { Link } from "react-router-dom";
const PageNotFound = () => (
  <div>
    <p>
      404 - PAGE NOT FOUND <Link to="/">GO BACK</Link>
    </p>
  </div>
);

export default PageNotFound;
2;
