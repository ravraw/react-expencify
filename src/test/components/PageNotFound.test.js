import React from "react";
import { Link } from "react-router-dom";
import { shallow } from "enzyme";

import PageNotFound from "../../components/PageNotFound";

test("Should render PageNotFound ", () => {
  const wrapper = shallow(<PageNotFound />);
  expect(wrapper).toMatchSnapshot();
});
