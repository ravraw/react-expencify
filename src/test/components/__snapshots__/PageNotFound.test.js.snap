// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render PageNotFound  1`] = `
<div>
  <p>
    404 - PAGE NOT FOUND 
    <Link
      replace={false}
      to="/"
    >
      GO BACK
    </Link>
  </p>
</div>
`;
